docs/pages/guides/using-firebase.mdx

```diff
@@ -94,7 +94,7 @@ You do not have to install other plugins or configurations to use Firebase JS SD
 
 Firebase version 9 provides a modular API. You can directly import any service you want to use from the `firebase` package. For example, if you want to use an authentication service in your project, you can import the `auth` module from the `firebase/auth` package.
 
-> Using Firebase Authentication? See [the guide for setting up persistence to keep users logged in between reloads](https://expo.fyi/firebase-js-auth-setup).
+> Using Firebase Authentication with SDK `v9.x.x` or below? See [the guide for setting up persistence to keep users logged in between reloads](https://expo.fyi/firebase-js-auth-setup).
```

Comment:
```suggestion
> Using Firebase Authentication with version 9 or below? See [the guide for setting up persistence to keep users logged in between reloads](https://expo.fyi/firebase-js-auth-setup).
```

Minor nit: We often use "SDK" to refer Expo SDK in our docs. Changed the above phrasing to explicitly say version number.